README.md

```diff
@@ -62,12 +67,20 @@ reflekt {
     // Use only DependencyHandlers which have canBeResolve = True
     // Note: Reflekt works only with kt files from libraries
     librariesToIntrospect = listOf("io.kotless:kotless-dsl:0.1.6")
+    // List of libraries, functions from which will be used 
+    // in the <filter> condition in SmartReflekt
+    // Use only DependencyHandlers which are <compileClasspath>
+    dependencyJar = emptyList()
```

Comment:
Now we don't have option `dependencyJar` and I'm not sure that we really need something like this.
In my opinion, the need to manage another list of copy-pasted dependencies with strict restrictions makes user experience much worse, and I don't think that scanning whole compileClasspath is bottleneck of compilation time.
What do you think?

Reply:
it is my mistake, sorry. thanks a lot!